### PR TITLE
VA reports for releases and release groups

### PR DESCRIPTION
**Pull request by gentlecat**
*http://tickets.musicbrainz.org/browse/MBS-2255*

### CODE REVIEW

File: lib/MusicBrainz/Server/Report/ReleasesWithoutVACredit.pm [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+with 'MusicBrainz::Server::Report::ReleaseReport';
+
+sub query {
+    "
+        SELECT
+            r.id AS release_id,
+            row_number() OVER (ORDER BY r.artist_credit, r.name)
+        FROM release r
+        JOIN artist_credit_name acn on acn.artist_credit = r.artist_credit
+        WHERE acn.artist = $VARTIST_ID AND acn.name != 'Various Artists'
```
Comment (mwiencek) [inline lib/MusicBrainz/Server/Report/ReleasesWithoutVACredit.pm]: It was mentioned on IRC that the release reports should probably also handle tracks, i.e. find tracks that meet those conditions, get the release via track -> medium -> release, and `UNION` the two queries.

File: lib/MusicBrainz/Server/Report/ReleaseGroupsWithoutVACredit.pm [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+sub template
+{
+    return 'report/release_groups_without_va_credit.tt';
+}
+
+__PACKAGE__->meta->make_immutable;
+no Moose;
+1;
+
+=head1 COPYRIGHT
```
Comment (mwiencek) [inline lib/MusicBrainz/Server/Report/ReleaseGroupsWithoutVACredit.pm]: A while ago we decided on a shorter copyright clause to use, just haven't bothered updating any of the old ones. The current ones are good too, obviously, they just take up more space. :)

```
=head1 COPYRIGHT

This file is part of MusicBrainz, the open internet music database.
Copyright (C) 2015 MetaBrainz Foundation
Licensed under the GPL version 2, or (at your option) any later version:
http://www.gnu.org/licenses/gpl-2.0.txt

=cut
```

File: lib/MusicBrainz/Server/Report/ReleaseGroupsWithoutVACredit.pm [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+        SELECT
+            rg.id AS release_group_id,
+            row_number() OVER (ORDER BY rg.artist_credit, rg.name)
+        FROM release_group rg
+        JOIN artist_credit_name acn on acn.artist_credit = rg.artist_credit
+        WHERE acn.artist = $VARTIST_ID AND acn.name != 'Various Artists'
+    ";
+}
+
+sub template
```
Comment (mwiencek) [inline lib/MusicBrainz/Server/Report/ReleaseGroupsWithoutVACredit.pm]: We also decided to always put the opening brace on the same line.

**Comment by mwiencek** [comment]
:rocket: ship

File: lib/MusicBrainz/Server/Report/ReleaseGroupsWithoutVACredit.pm (hunk 2 above)
Comment (gentlecat) [inline lib/MusicBrainz/Server/Report/ReleaseGroupsWithoutVACredit.pm]: Ah. I just copy pasted one of old reports. Let me update it.

**Pull request merged**